ansible/configs/ocp4-cluster/default_vars_azure.yml

```diff
@@ -50,3 +50,7 @@ master_instance_type: Standard_D8s_v3
 # Machine Type for worker nodes
 worker_instance_type: Standard_D4s_v3
 
+# admin keys. Azure specific. Other platforms get these from the bastion roles
+opentlc_admin_pub_keys:
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3Avw03Dmh1R2QWQ4CV7JgEsXnHQjNhfppD5aZmh0q/64p6lW+2oNKTT7fVQcrsdmlJwrMd5apkUGrOcq0hHXQMEVZEKUmEjko2BqD5A9/zNX7apObW88bFFfgxc91lOT+e+wfCFsrr3b2SJ3+KL6nTBJV7Lf46i6z86vhiDPjqL7U9kTS+bK9ldU20vpn8h+ZAIaiafVWfjihUjhNpcUY46klixV1YcAkBGCbE+YR6RAAc6vWy0zB3YJnTUl9OFt213ofi1qjuWKVMmOxORxPKB4/JQ+hfAsCMysoVFnFYs10dWxaySK63OgY9uLNyaIwkEaVVIfcViRVm0DZfoNH gucore
```

Comment:
We can probably remove GCs key from every default we have...

Reply:
Yeah.